tapchannel/records.go

```diff
@@ -384,6 +386,17 @@ func (c *Commitment) Decode(r io.Reader) error {
 	return tlvStream.Decode(r)
 }
 
+// Bytes returns the serialized Commitment record.
+func (c *Commitment) Bytes() []byte {
```

Comment:
Lots of rebase artifacts in this file, you can probably get rid of all the diffs except for this method.

Reply:
Yeah not sure what happened here, will clean it up.